src/components/Select/Select.tsx

```diff
@@ -11,44 +10,54 @@ export interface SelectProps {
     blockName?: string;
     /** ClassName you can add in addition to 'select' */
     className?: string;
+    children?: React.ReactNode;
     /** When true, disables the select */
     disabled?: boolean;
     /** ID of associated HelperText */
     helperTextId?: string;
     /** ID that other components can cross reference for accessibility purposes */
     id?: string;
-    /** ID of associated label */
-    labelId?: string;
     /** Attribute indicating that an option with a non-empty string value must be selected */
     isRequired: boolean;
-    onBlur: (event: React.FormEvent) => void;
-    /** Passes selects' current value to the React state handler */
-    onChange: (event: React.FormEvent) => void;
+    /** ID of associated label */
+    labelId?: string;
     /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
     modifiers?: string[];
+    /** The name of the select element to use in form submission */
+    name: string;
+    /** Passes selects' current value to the React state handler */
+    onBlur?: (event: React.FormEvent) => void;
+    /** Passes selects' current value to the React state handler */
+    onChange?: (event: React.FormEvent) => void;
     /** Sets whatever string you pass it as the default selected */
     selectedOption?: string;
 }
 
-export default class Select extends React.Component<
-    SelectProps,
-    { selectedOption: string }
-> {
-    constructor(props: SelectProps) {
-        super(props);
-        this.state = { selectedOption: props.selectedOption };
-        this.onSelectChange.bind(this);
-    }
-
-    onSelectChange(event: React.FormEvent, additionalChange: Function) {
-        let target = event.target as HTMLSelectElement;
-        if (target) {
-            this.setState({ selectedOption: target.value });
-        }
-        additionalChange(event);
-    }
-
-    render() {
+/**
+ * Select
+ * A component that renders a `select` DOM element along with its `option`
+ * children.
+ */
+const Select = React.forwardRef<HTMLSelectElement, SelectProps>(
```

Comment:
Is there an argument to just go through all the form elements in one go and enable ref forwarding in all of them?  Or is it ok to just add them as we need them?  (this isn't a breaking change so I guess there's not an urgent need to get them all in at once ... ) Which are the components that this would affect?

Reply:
Right now, the `Pagination` and `SearchBar` components import `Select` but this update doesn't affect it. I could go through the rest of the components but since the PR is for the select element, I just focused on that. I figured if more components need to be updated that can be a different PR.

Reply:
Oh I mean like would you recommend a design system ticket that's like "go through all the components and add refs to places that need it" or is it ok to do ad-hoc

Reply:
I think it might be good to do ad-hoc because being able to use and test it in other components makes it easier to know that it's working as it should. I don't want to introduce a bug updating components and not testing it outside of the repo.